universal-application-tool-0.0.1/app/services/applicant/question/RepeaterQuestion.java

```diff
@@ -1,20 +1,26 @@
 package services.applicant.question;
 
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import services.applicant.ValidationErrorMessage;
 import services.question.types.QuestionType;
 import services.question.types.RepeaterQuestionDefinition;
 
 public class RepeaterQuestion implements PresentsErrors {
 
   private final ApplicantQuestion applicantQuestion;
 
+  // TODO(#859): make this admin-configurable
+  private final String PLACEHOLDER = "placeholder";
```

Comment:
Since this isn't implemented yet (and it's possible we won't get to it), I think it'd be better to leave this as an empty string for now. The string "placeholder" doesn't add any value and might just be more confusing than the empty string.

Reply:
fair enough!